packages/client/src/components/pages/home-page/home-page.tsx

```diff
@@ -138,20 +135,10 @@ export const HomePage = () => {
     void (async () => {
       for (const workspace of workspaces) {
         await dispatch(Workspace.fetchDashboards(orgId, workspace.id));
-      }
-      
-      try {
-        await dispatch(OrphanedRecordActions.fetchPage({
-          orgId,
-          query: {
-            page: '',
-            limit: '',
-            unit: undefined,
-          },
-        }));
-      } catch (error) {
-        void dispatch(Modal.alert('Get Orphaned Records', formatErrorMessage(error, 'Failed to get orphaned records')));
-      }
+      }    
+      const orcResponse = await OrphanedRecordClient.getOrphanedRecordsCount(orgId);
+      setOrphanedRecordCount(orcResponse.count);
```

Comment:
This data is actually already in the redux store, and can be refreshed via an action. So you shouldn't need to store it as state in the home page. Here's an example of what that would look like...

```typescript
void dispatch(OrphanedRecordActions.fetchCount({ orgId: orgId! }));
```